src/test/java/uk/gov/companieshouse/web/accounts/transformer/smallfull/CurrentAssetsTransformerImplTests.java

```diff
@@ -0,0 +1,349 @@
+        assertNotNull(balanceSheet.getCurrentAssets().getCashAtBankAndInHand().getPreviousAmount());
+        assertNotNull(balanceSheet.getCurrentAssets().getDebtors().getPreviousAmount());
+        assertNotNull(balanceSheet.getCurrentAssets().getStocks().getPreviousAmount());
+        assertNotNull(balanceSheet.getCurrentAssets().getPreviousTotal());
+    }
+
+    private void assertCurrentPeriodValuesCorrect(BalanceSheet balanceSheet) {
+        assertEquals(CURRENT_CASH_IN_BANK_AND_IN_HAND, balanceSheet.getCurrentAssets().getCashAtBankAndInHand().getCurrentAmount());
+        assertEquals(CURRENT_DEBTORS, balanceSheet.getCurrentAssets().getDebtors().getCurrentAmount());
+        assertEquals(CURRENT_STOCKS, balanceSheet.getCurrentAssets().getStocks().getCurrentAmount());
+        assertEquals(CURRENT_TOTAL, balanceSheet.getCurrentAssets().getCurrentTotal());
+    }
+
+    private void assertPreviousPeriodValuesCorrect(BalanceSheet balanceSheet) {
+        assertEquals(PREVIOUS_CASH_IN_BANK_AND_IN_HAND, balanceSheet.getCurrentAssets().getCashAtBankAndInHand().getPreviousAmount());
+        assertEquals(PREVIOUS_DEBTORS, balanceSheet.getCurrentAssets().getDebtors().getPreviousAmount());
+        assertEquals(PREVIOUS_STOCKS, balanceSheet.getCurrentAssets().getStocks().getPreviousAmount());
+        assertEquals(PREVIOUS_TOTAL, balanceSheet.getCurrentAssets().getPreviousTotal());
+    }
+
+    private void assertApiModelFieldsNotNull(BalanceSheetApi balanceSheetApi) {
+        assertNotNull(balanceSheetApi.getCurrentAssetsApi());
+        assertNotNull(balanceSheetApi.getCurrentAssetsApi().getCashAtBankAndInHand());
+        assertNotNull(balanceSheetApi.getCurrentAssetsApi().getDebtors());
+        assertNotNull(balanceSheetApi.getCurrentAssetsApi().getStocks());
+        assertNotNull(balanceSheetApi.getCurrentAssetsApi().getTotal());
+    }
+
+    private void assertCurrentPeriodApiModelValuesCorrect(BalanceSheetApi balanceSheetApi) {
+        assertEquals(CURRENT_CASH_IN_BANK_AND_IN_HAND, balanceSheetApi.getCurrentAssetsApi().getCashAtBankAndInHand());
+        assertEquals(CURRENT_DEBTORS, balanceSheetApi.getCurrentAssetsApi().getDebtors());
+        assertEquals(CURRENT_STOCKS, balanceSheetApi.getCurrentAssetsApi().getStocks());
+        assertEquals(CURRENT_TOTAL, balanceSheetApi.getCurrentAssetsApi().getTotal());
+    }
+
+    private void assertPreviousPeriodApiModelValuesCorrect(BalanceSheetApi balanceSheetApi) {
+        assertEquals(PREVIOUS_CASH_IN_BANK_AND_IN_HAND, balanceSheetApi.getCurrentAssetsApi().getCashAtBankAndInHand());
+        assertEquals(PREVIOUS_DEBTORS, balanceSheetApi.getCurrentAssetsApi().getDebtors());
+        assertEquals(PREVIOUS_STOCKS, balanceSheetApi.getCurrentAssetsApi().getStocks());
+        assertEquals(PREVIOUS_TOTAL, balanceSheetApi.getCurrentAssetsApi().getTotal());
+    }
+
+    private void assertOtherModelFieldsNull(BalanceSheet balanceSheet) {
```

Comment:
You mentioned naming, maybe something like `assertNonCurrentAssetsFieldsNull`? I'm not fussy either way, more of a suggestion